sensorlib/src/noise_models/AbstractNoiseModel.py

```diff
@@ -53,3 +53,45 @@ def apply_list_inclusion_noise(self, object_list):
         :return: List of DetectedObject objects with noise applied.
         """
         pass
+
+    @abstractmethod
+    def apply_position_covariance(self, object_list):
```

Comment:
Should be renamed to `apply_position_covariance_noise` to match the naming scheme of the other functions.